dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/WorkflowExecuteRunnable.java

```diff
@@ -805,6 +809,65 @@ public void checkSerialProcess(ProcessDefinition processDefinition) {
         commandService.createCommand(command);
     }
 
+    private void logTaskInstanceInDetail(TaskInstance taskInstance) {
```

Comment:
Please move this kind of method to utils

Reply:
done.